src/java/htsjdk/samtools/util/Locatable.java

```diff
@@ -22,4 +22,63 @@
      * @return 1-based closed-ended position, undefined if getContig() == null
      */
     int getEnd();
+
+
+    /**
+    * @return the 0-based start position (from the GA4GH spec).
+    */
+    default long getGA4GHStart() {return this.getStart() - 1; }
```

Comment:
Personally I'd prefer to keep them out.  A compromise might be to add a little `ZeroBasedOpenEnd` class and have a `toZeroBasedOpenEnd()` method or something, but even that I'm not keen on.